frontend/src/concepts/analyticsTracking/useTrackUser.ts

```diff
@@ -26,9 +29,11 @@ export const useTrackUser = (username?: string): [IdentifyEventProperties, boole
       return aId;
     };
 
-    computeAnonymousUserId().then((val) => {
-      setAnonymousId(val);
-    });
+    if (!userID) {
+      computeAnonymousUserId().then((val) => {
+        setAnonymousId(val);
+      });
+    }
```

Comment:
If you never compute the anonymousID, then the user props will never be considered loaded according to the last line:
```
return [props, acLoaded && !!anonymousId];
```

Shouldn't the anonymousId be set to the incoming userID instead?

Reply:
Done